scripts/spark_eval/copy_exome_to_hdfs_on_gcs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Copy exome data to HDFS on GCS
+
+${GATK_HOME:-../..}/gatk-launch ParallelCopyGCSDirectoryIntoHDFSSpark \
```

Comment:
Should this be explicitly setting the number of executors?

Reply:
I don't think it needs to - at least it works fine without.